cmd/protoc-gen-openapi/generator/openapi-v3.go

```diff
@@ -891,13 +887,75 @@ func (g *OpenAPIv3Generator) addSchemasToDocumentV3(d *v3.Document, messages []*
 			contains(g.generatedSchemas, typeName) {
 			continue
 		}
+
 		g.generatedSchemas = append(g.generatedSchemas, typeName)
+
+		// google.protobuf.Value is handled like a special value when doing transcoding.
+		// It's interpreted as a "catch all" JSON value, that can be anything.
+		if message.Desc != nil && message.Desc.FullName() == "google.protobuf.Value" {
```

Comment:
Is this test so that `AnyJSONValue` is only added when it is needed? Is there something that ensures that it's not added to the schema list more than once?

Reply:
Exactly 👍
As far as I can remember, due to the `AnyJSONValue` naming, the existing tests will ensure that it is not added more than once.

Reply:
I'm looking through the code right now, to ensure that it doesn't get added more than once.

Reply:
It is covered in the test and it only get's added once 👍

Reply:
Great! I thought so, but I hadn't figured out how that worked.

Reply:
openapi_json.yml line 184 - 195 has a couple of $refs and on line 211 -221, it is declared once.